components/supervisor/frontend/webpack.config.js

```diff
@@ -10,14 +10,14 @@ const webpack = require("webpack");
 const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 
+/**@type {import('webpack').Configuration}*/
 module.exports = {
     target: "web",
     entry: {
         main: path.resolve(__dirname, "lib/index.js"),
     },
     output: {
         filename: "[name].js",
-        chunkFilename: "[name].js",
```

Comment:
why do we need to change this?

Reply:
it's not relevant we always generate one file